SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -576,10 +576,20 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 
 	if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
 	{
-		UE_LOG(LogSpatialActorChannel, Error, TEXT("Failed to create entity for actor %s: %s"), *Actor->GetName(), UTF8_TO_TCHAR(Op.message));
-		Sender->SendCreateEntityRequest(this, GetPlayerWorkerId());
+		// Temporary hack to avoid failure to reserve entities due to timeout on large maps
```

Comment:
I understand the original codepath can cause a problem if creation of the same entity gets requested from different places somehow. Suddenly you'd start getting "Entity exists" error because another request has succeeded. And in that case you indeed shouldn't be retrying as retrying is silly and will never succeed.

But if that's the case, we should identify how that is happening. In theory there shouldn't be more than one request in flight for the same entity.

Reply:
I agree, 360 was committed as a quick-fix hack.  As this issue is blocking Production milestones, and as the hack workaround was already checked into Master, I figured I would at least submit a fix for the fix, until someone has bandwidth to look into the real cause.

Reply:
Is there a bug to track the issue I described above?

Reply:
I'm not sure.  I think Sahil authored 360.